DataFormats/TrackingRecHitSoA/src/classes.h

```diff
@@ -0,0 +1,11 @@
+#ifndef DataFormats_TrackingRecHitSoA_src_classes_h
+#define DataFormats_TrackingRecHitSoA_src_classes_h
+
+#include "DataFormats/Common/interface/Wrapper.h"
+#include "Geometry/CommonTopologies/interface/SimplePixelTopology.h"
+#include "DataFormats/TrackingRecHitSoA/interface/TrackingRecHitsSoA.h"
+#include "DataFormats/TrackingRecHitSoA/interface/TrackingRecHitsHost.h"
+
+using namespace pixelTopology;
```

Comment:
Is this line needed?

Reply:
Actually, yes.
Without we get errors (no idea why) like
```
>> Building LCG reflex dict from header file src/DataFormats/TrackingRecHitSoA/src/classes.h
Error in <TInterpreter::InspectMembers>: Cannot find TClass for base class PortableHostCollection<TrackingRecHitSoA<pixelTopology::HIonPhase1>::Layout<128,false> >
Error in <TInterpreter::InspectMembers>: Cannot find TClass for base class PortableHostCollection<TrackingRecHitSoA<pixelTopology::Phase2>::Layout<128,false> >
Error in <TInterpreter::InspectMembers>: Cannot find TClass for base class PortableHostCollection<TrackingRecHitSoA<pixelTopology::Phase1>::Layout<128,false> >
>> Compiling LCG dictionary: tmp/el8_amd64_gcc12/src/DataFormats/TrackingRecHitSoA/src/DataFormatsTrackingRecHitSoA/a/DataFormatsTrackingRecHitSoA_xr.cc
DataFormatsTrackingRecHitSoA/a/DataFormatsTrackingRecHitSoA_xr.cc:205:105: error: 'HIonPhase1' was not declared in this scope; did you mean 'pixelTopology::HIonPhase1'?
  205 |    static TGenericClassInfo *GenerateInitInstanceLocal(const ::PortableHostCollection<TrackingRecHitSoA<HIonPhase1>::Layout<128,false> >*)
      |                                                                                                         ^~~~~~~~~~
      |                                                                                                         pixelTopology::HIonPhase1
```

Reply:
I noticed the [compilation log](https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-7818e6/37025/build.log) has
```
Error in <TInterpreter::InspectMembers>: Cannot find TClass for base class PortableDeviceCollection<TrackingRecHitSoA<pixelTopology::Phase2>::Layout<128,false>,alpaka::DevUniformCudaHipRt<alpaka::ApiHipRt>,void>
Error in <TInterpreter::InspectMembers>: Cannot find TClass for base class PortableDeviceCollection<TrackingRecHitSoA<pixelTopology::Phase1>::Layout<128,false>,alpaka::DevUniformCudaHipRt<alpaka::ApiHipRt>,void>
```
Given that these are only for `PortableDeviceCollection` for HIP (no idea why `HIonPhase1` is not there), could it be related to `using namespace pixelTopology;` not being in the `classes_rocm.h`?

Maybe at this point it would be best to include the `using namespace pixelTopology;` in all of the `classes*.h` files, and investigate these afterwards.

Reply:
> (no idea why `HIonPhase1` is not there),

That is probably explained by the dictionaries for the `pixelTopology::HIonPhase1` instantiations not being declared in `classes_{cuda,rocm}_def.xml` files.

Reply:
> Maybe at this point it would be best to include the `using namespace pixelTopology;` in all of the `classes*.h` files, and investigate these afterwards.

OK, I've added it back everywhere.